libraries/AP_MSP/msp_osd.h

```diff
@@ -0,0 +1,143 @@
+#pragma once
+
+#include <stdint.h>
+
+#if HAL_MSP_ENABLED
+
+#define OSD_FLAGS_OSD_FEATURE           (1 << 0)
+
+namespace MSP
+{
+typedef enum {
```

Comment:
`enum class`

Reply:
not easy, I use them to access an array, again explicit casting would be required